README.md

```diff
@@ -1,11 +1,12 @@
-# Similar Words
-Discover which words are most similar to each other. Trained on billions of tweets
+# Conversation NLP
```

Comment:
`Conversation NLP` is way too general. The name is so general that it could be a valid name for any app that has anything to do with text; the fact that natural language is being processed doesn't tell you anything about what value you get from the processing. 

Here are some suggestions for a new name to replace `Similar Words`:
  * `Associated Words`
  * `Twitter Context`
  * `Twitter Keywords`
  * `Twitter Tags`
  * `Social Context`
  * `Keyword Finder`
  * `Tag Finder`
  * `Social Tags`
  * `Social Keywords`
  * `Related Words`
  * `Related Keywords`
  * `Related Tags`
  * `Companion Words`